### ISSUE TITLE
Attempting to edit item from admin page leads to broken, offscreen form

### ISSUE THREAD

**Issue by jpaddison3** [LessWrong2/Lesswrong2#877]
Item being either a user or a post.
![image](https://user-images.githubusercontent.com/10352319/44726985-d9df2700-aad0-11e8-8d05-6286e327f91f.png)

**Comment by Discordius** [comment]
Yeah, I don't think we ever really use the admin page, so not too surprised. My guess is that it's some kind of styling bug after we removed all of bootstrap.
